front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -147,7 +151,27 @@ const FullscreenModal = ({
         contentBgColor={contentBgColor}
       >
         <StyledFocusOn autoFocus>
-          {topBar}
+          {modalTitle && (
+            <Box
+              bgColor={colors.white}
+              borderBottom={`1px solid ${colors.grey300}`}
+              px="16px"
```

Comment:
Default styles that make sense. We can add props for custom styles later if needed.